### ISSUE TITLE
Annotation file

### ISSUE THREAD

**Issue by wangge-1996** [JiaqiLiZju/Nvwa#7]
Hello, I think this is a very good study. In code" python ../../2_propare_datasets.py train_test_split $Onehot $Label $Annotation Dataset.train_test.h5", $Annotation is a txt file, which is not explained in the related code how to build, so you want to know how to generate this file.

**Comment by JiaqiLi1024** [comment]
You can access our dataset through the following link: https://bis.zju.edu.cn/nvwa/dataset.html. If you're planning to create your own datasets and models, your annotation file should minimally include three columns: `cell_id`, `cell_type`, and `cell_lineage`. Alternatively, you can refer to the data structure in `Dataset.*_train_test.h5` for guidance.

**Comment by wangge-1996** [comment]
Thank you! The problem has been solved.

I still have a problem: Single-cell data from closely related species can be used for data enhancement, but how can it be paired with gene expression tags in the training set?

**Comment by JiaqiLi1024** [comment]
For a specific gene, we firstly identified orthologs genes from other close-related species, and paired those orthologs gene sequences with the corresponding gene expression tags. Please note that only training set should be augmented.

**Comment by wangge-1996** [comment]
I see. Thank you for answering my confusion.

**Comment by JiaqiLi1024** [comment]
cool, have fun with it.
